realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/ProcessCommitTests.java

```diff
@@ -109,14 +108,13 @@ public void onChange(RealmResults<ProcessInfo> element) {
         }
     }
 
-    // FIXME: Ignore for now. They do still not work. It might be caused by two processes each creating
     // a Sync Client, but it needs to be investigated.
     //TODO send string from service and match
     //     replicate integration tests from Cocoa
     //     add gradle task to start the sh script automatically (create pid file, ==> run or kill existing process
     //     check the requirement for the issue again
     @Test
-    @Ignore
+    @Ignore("Ignore for now. Still doesn't work. It might be caused by two processes each creating")
```

Comment:
Exactly.  Instead of commenting it out, it is now explicitly ignored.  I have no idea what the problem is.